code/modules/mob/living/carbon/resist.dm

```diff
@@ -79,27 +79,58 @@
 	//If you are handcuffed with actual handcuffs... Well what do I know, maybe someone will want to handcuff you with toilet paper in the future...
 	if(istype(HC))
 		breakouttime = HC.breakouttime
-		displaytime = breakouttime / 600 //Minutes
+	
+	if(buckled)
+		breakouttime += 600 //If you are buckled, it takes a minute longer, but you are unbuckled and uncuffed instantly
+	
+	displaytime = breakouttime / 600 //Minutes
 
 	var/mob/living/carbon/human/H = src
 	if(istype(H) && H.gloves && istype(H.gloves,/obj/item/clothing/gloves/rig))
 		breakouttime /= 2
 		displaytime /= 2
 
-	visible_message(
-		"<span class='danger'>\The [src] attempts to remove \the [HC]!</span>",
-		"<span class='warning'>You attempt to remove \the [HC]. (This will take around [displaytime] minutes and you need to stand still)</span>"
-		)
+	//Check if the user is trying to violently remove the handcuffs
+	var/violent_removal = 0
+	if(a_intent == I_HURT)
+		violent_removal = 1
+		breakouttime = rand(450,600)
+		visible_message(
+			"<span class='danger'>\The [src] attempts to break out of \the [HC]!</span>",
+			"<span class='warning'>You attempt to break out of \the [HC]. (This will take around 1 minute and you need to stand still)</span>"
+			)
+	else
+		visible_message(
+			"<span class='danger'>\The [src] attempts to slip out of \the [HC]!</span>",
+			"<span class='warning'>You attempt to slip out of \the [HC]. (This will take around [displaytime] minutes and you need to stand still)</span>"
+			)
 
 	if(do_after(src, breakouttime))
-		if(!handcuffed || buckled)
+		if(!handcuffed)
 			return
-		visible_message(
-			"<span class='danger'>\The [src] manages to remove \the [handcuffed]!</span>",
-			"<span class='notice'>You successfully remove \the [handcuffed].</span>"
-			)
+
+		var/buckle_message_user = ""
+		var/buckle_message_other = ""
+		if(buckled) //If the person is buckled, also unbuckle the person
+			buckle_message_user = " and unbuckle yourself"
+			buckle_message_other = " and to unbuckle themself"
+			buckled.user_unbuckle_mob(src)
+
+		if(violent_removal)
+			var/obj/item/organ/external/E = H.get_organ(pick("l_arm","r_arm"))
+			E.dislocate(1)

```

Comment:
Can `get_organ` return `null?` At which point, do a `if (!E) return` dealie before this.